index.js

```diff
@@ -3,26 +3,34 @@
 var gju = require('geojson-utils');
 
 var leafletPip = {
-    bassackwards: false,
-    pointInLayer: function(p, layer, first) {
-        if (p instanceof L.LatLng) p = [p.lng, p.lat];
-        else if (leafletPip.bassackwards) p = p.concat().reverse();
+  bassackwards: false,
+  pointInLayer: function(p, layer, first) {
+    if (p instanceof L.LatLng) p = [p.lng, p.lat];
+    else if (leafletPip.bassackwards) p = p.concat().reverse();
 
-        var results = [];
+    var results = [];
 
-        layer.eachLayer(function(l) {
-            if (first && results.length) return;
-            if ((l instanceof L.MultiPolygon ||
-                 l instanceof L.Polygon) &&
-                gju.pointInPolygon({
-                    type: 'Point',
-                    coordinates: p
-                }, l.toGeoJSON().geometry)) {
-                results.push(l);
-            }
-        });
-        return results;
-    }
+    layer.eachLayer(function(l) {
+      if (first && results.length) return;
+
+      if (isPoly(l) &&
+        gju.pointInPolygon({
+          type:        'Point',
```

Comment:
My IDE is setup to follow [airbnb JS style guide](https://github.com/airbnb/javascript#whitespace).  Looks like you use 4 spaces instead of 2. I can switch the PR to use 4 if you'd like.